server/src/main/java/io/unitycatalog/server/security/SecurityContext.java

```diff
@@ -0,0 +1,126 @@
+package io.unitycatalog.server.security;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.algorithms.Algorithm;
+import com.auth0.jwt.interfaces.DecodedJWT;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.unitycatalog.server.UnityCatalogServer;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.security.interfaces.RSAPrivateKey;
+import java.security.interfaces.RSAPublicKey;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import lombok.Getter;
+import lombok.SneakyThrows;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class SecurityContext {
+
+  public interface Issuers {
```

Comment:
Nit: why this instead of an enum?

Reply:
Token issuers are typically URIs and expressed as a String in the JWT claim, so I made the internal ISSUER data type match.